### DISCUSSION TITLE
Проблема с Accepts mime-типом в поле файл

### DISCUSSION THREAD

**Discussion by DaveMaks** [totumonline/totum-ru-issues-and-discussions#321] [ОШБК!] [answered]
Доброго времени суток, 
В типе поля файл, в параметре mime-тип не правильно транслируется в html
![image](https://user-images.githubusercontent.com/54938636/215269138-91839d77-57df-4854-a5c2-806828e246eb.png)
![image](https://user-images.githubusercontent.com/54938636/215269157-6f11ec7d-22f6-4f7e-8723-2dbd4e450210.png)
они должны быть через запятую, в результате при нажатии добавить файл не видит файлов, а если указать через запятую то не работает валидация на сервере.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. Мы чинили там что-то какое-то время назад.

2. Что имеется в виду под «Валидация на сервере»?

**Reply by ttm-alexeypolunin** [reply]
А понял, смотрел в EN документацию, а она отстает сильно.

Да — это мы фигню сделали, надо через `,`. Исправим в пн-вт.

**Reply by ttm-alexeypolunin** [reply]
Исправили
